test_rclcpp/test/node_check_names.cpp

```diff
@@ -40,16 +43,23 @@ int main(int argc, char ** argv)
   int counter = 0;
   const std::chrono::steady_clock::time_point max_runtime =
     std::chrono::steady_clock::now() + std::chrono::seconds(10);
-  std::string node_to_look_for("node_with_name");
+
   while (rclcpp::ok()) {
-    printf("\n");
-    auto names = node->get_node_graph_interface()->get_node_names();
+    auto names_namespaces = node->get_node_graph_interface()->get_node_names_and_namespaces();
```

Comment:
Is `get_node_names_and_namespaces()` supposed to be `get_node_names()`?

Reply:
I changed it to get_node_names_and_namespaces since the only thing that these functions are interested in is the name, not a fully qualified name. As such, I figured it's easier to grab the name from a tuple than extract from a string.